### ISSUE TITLE
Unable to create NFS policy with multiple rules

### ISSUE THREAD

**Issue by MilesTails** [Pure-Storage-Ansible/FlashArray-Collection#264]
**Describe the bug**
Unable to create NFS policy with multiple rules for certain combinations. Error claims that the second rule access permissions are not the same as the first rule. Confirmed that we are able to create the rule via the gui

**To Reproduce**
Data Model:
```yaml
fa_nfs_policies:
  - name: "multiple-rules-nfs"
    rules:
      - client: "10.0.1.0/24"
        nfs_access: "no-root-squash"
        nfs_permission: "rw"
      - client: "10.0.2.0/24"
        nfs_access: "no-root-squash"
        nfs_permission: "rw"
```

Ansible task:
```yaml
- name: Create NFS policies
  purestorage.flasharray.purefa_policy:
    name: "{{item.0.name}}"
    policy: nfs
    nfs_access: "{{item.1.nfs_access | default('no-root-squash')}}"
    nfs_permission: "{{item.1.nfs_permission | default('rw')}}"
    client: "{{item.1.client | default('*')}}"
    state: present
    fa_url: "{{ fa_url }}"
    api_token: "{{ fa_token }}"
  loop: "{{ fa_nfs_policies | subelements('rules') }}"
```

3. On run receive error: 
`"msg": "Failed to create new rule for NFS policy multiple-rules-nfs. Error: Access options must be the same for all NFS export policy rules.",
`

Fails on: 
- root-squash + ro
- no-root-squash + ro
- no-root-squash + rw

Succeeds on: 
- root-squash + rw

**Expected behavior**
It creates the second rule.

**Environment**
Ansible 2.10.2
Collection v1.12.1
Purity//FA - 6.1.10

**Comment by sdodsley** [comment]
@MilesTails Please see PR #265 and confirm whether this resolves your issue.

**Comment by MilesTails** [comment]
@sdodsley patched in prod, working like a charm. Thanks!

**Comment by sdodsley** [comment]
Resolved by #265
